tests/data_schemas.py

```diff
@@ -39,12 +39,13 @@ def _recurse_defaults(d: dict):
         _recurse_defaults(values["defaults"])
         return values
 
-        # import ipdb; ipdb.set_trace()
-        # return super().validate(value)
-
 
 if __name__ == "__main__":
-    data = Path(__file__).parent / "voting_systems.json"
+    data = (
+        Path(__file__).parent.parent
+        / "uk_election_ids"
+        / "data"
+        / "voting_systems.json"
+    )
     schema = VotingSystemSchema.parse_file(data)
     VotingSystemSchema.validate(schema)
```

Comment:
Yeah good question. In that sense, this file isn't really a "test". Essentially, `validate()` will either do nothing if the file is valid (in which case this script will exit with code 0) or it will throw an exception, in which case this will exit with a non-zero status code causing the build to fail. So in that sense I figure it is a useful script to run in CI but it is not strictly a "test".

Reply:
Ah I see that makes sense.